docs/components/console/manage-organization/enable-alpha-features.md

```diff
@@ -38,3 +38,12 @@ Once you accept the alpha terms, you can enable and disable any features you wou
 - In the **Settings** tab, toggle the switch under **Status** to enable and disable the feature.
 - Click **View docs** under **Documentation** to learn more about the feature.
 - Admins can know when someone accepts the alpha terms and when features are enabled or disabled under the **Activity** tab in **Organization Management**.
+
+### Enable AI-powered features
+
+To use AI-powered alpha features, the **Terms for AI Usage** must be agreed to before the toggle will appear in the **Status** column.
+
+- In the **Settings** tab, click the **Opt-in to enable** link under **Status**.
+- A dialog will appear containing the **Terms for AI Usage**. You must read to the end of the terms before accepting.
+- To agree to the terms, select the box labeled **"I understand and agree to the Terms for AI Usage"**.
```

Comment:
Are there quotes in the UI? If not, can likely remove from **I understand and agree to the Terms for AI Usage**. I realize this is a bit nit picky.